index.html

```diff
@@ -0,0 +1,204 @@
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Polska</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">România</a>
+        </p>
+      </div>
+      <div class="d-flex flex-wrap">
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">About US</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Corrections</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Contact Us</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Editorial Guidelines</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Advertise</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Copyright</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Terms & Conditions</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Privacy Policy</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Cookie Policy</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Terms of Sale</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Archive</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+        <p class="px-2 mb-1 text-secondary">
+          <a class="links text-decoration-none" href="#">Announcements</a>
+        </p>
+        <span class="d-sm-none d-md-block text-secondary"> | </span>
+      </div>
```

Comment:
Using the `<span>` and `<p>` tags to list these items is not advisable. It is ideal to use the `<ul><li>` tag to list these items.